Makefile

```diff
@@ -58,16 +61,12 @@ go-deps:
 
 install-tools:
 	GO111MODULE=on go build -o $(GOPATH)/bin/golangci-lint ./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint
-	GO111MODULE=on go build -o $(GOPATH)/bin/gosec ./vendor/github.com/securego/gosec/cmd/gosec
 
 # Run verification steps
 # Example:
 #    make verify
 verify: install-tools
 	golangci-lint run --build-tags=$(GOTAGS)
```

Comment:
does this by default invokes gosec or we need to add some flag to enable this?

Reply:
I've enabled it in `.golangci.yml` explicitly now, thank you!